Preserter mode does not work correctly when mode contains time in the import

**Describe the bug**
When there is this: `Library    Browser    presenter_mode={'color': 'white', 'duration': '5s'}`

Then there is failure
```python
   @property
    def get_presenter_mode(self) -> HighLightElement:
        mode: dict = {}
        if isinstance(self.library.presenter_mode, dict):
            mode = copy(self.library.presenter_mode)  # type: ignore
        duration = mode.get("duration", "2 seconds")
>       duration = timedelta(seconds=timestr_to_secs(duration))

duration   = datetime.timedelta(seconds=5)
mode       = {'color': 'white', 'duration': datetime.timedelta(seconds=5)}
self       = <Browser.base.librarycomponent.LibraryComponent object at 0x107d2c8b0>

Browser/base/librarycomponent.py:230: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

timestr = datetime.timedelta(seconds=5), round_to = 3
accept_plain_values = True

    def timestr_to_secs(timestr, round_to=3, accept_plain_values=True):
        """Parses time like '1h 10s', '01:00:10' or '42' and returns seconds."""
        if is_string(timestr) or is_number(timestr):
            if accept_plain_values:
                converters = [_number_to_secs, _timer_to_secs, _time_string_to_secs]
            else:
                converters = [_timer_to_secs, _time_string_to_secs]
            for converter in converters:
                secs = converter(timestr)
                if secs is not None:
                    return secs if round_to is None else round(secs, round_to)
>       raise ValueError("Invalid time string '%s'." % timestr)
E       ValueError: Invalid time string '0:00:05'.
````

--- a/Browser/base/librarycomponent.py
+++ b/Browser/base/librarycomponent.py
@@ -224,7 +224,9 @@ class LibraryComponent:
         mode: dict = {}
         if isinstance(self.library.presenter_mode, dict):
             mode = copy(self.library.presenter_mode)  # type: ignore
-        duration = timedelta(seconds=timestr_to_secs(mode.get("duration", "2 seconds")))
+        duration = mode.get("duration", "2 seconds")
+        if not isinstance(duration, timedelta):
+            duration = timedelta(seconds=timestr_to_secs(duration))
         width = mode.get("width", "2px")
         style = mode.get("style", "dotted")
         color = mode.get("color", "blue")

--- a/utest/approved_files/test_presenter_mode_conversion.test_presenter_mode_default.approved.txt
+++ b/utest/approved_files/test_presenter_mode_conversion.test_presenter_mode_default.approved.txt
@@ -1,0 +1,1 @@
+{'duration': datetime.timedelta(seconds=2), 'width': '2px', 'style': 'dotted', 'color': 'blue'}

--- a/utest/approved_files/test_presenter_mode_conversion.test_presenter_mode_duration_as_string.approved.txt
+++ b/utest/approved_files/test_presenter_mode_conversion.test_presenter_mode_duration_as_string.approved.txt
@@ -1,0 +1,1 @@
+{'duration': datetime.timedelta(seconds=4), 'width': '2px', 'style': 'dotted', 'color': 'white'}

--- a/utest/approved_files/test_presenter_mode_conversion.test_presenter_mode_duration_as_timedelta.approved.txt
+++ b/utest/approved_files/test_presenter_mode_conversion.test_presenter_mode_duration_as_timedelta.approved.txt
@@ -1,0 +1,1 @@
+{'duration': datetime.timedelta(seconds=5), 'width': '2px', 'style': 'dotted', 'color': 'black'}

--- a/utest/test_presenter_mode_conversion.py
+++ b/utest/test_presenter_mode_conversion.py
@@ -1,0 +1,30 @@
+from datetime import timedelta
+from unittest.mock import MagicMock, PropertyMock
+
+from approvaltests.approvals import verify  # type: ignore
+
+from Browser.base.librarycomponent import LibraryComponent
+
+
+def test_presenter_mode_default():
+    lib = MagicMock()
+    type(lib).presenter_mode = {}
+    ctx = LibraryComponent(lib)
+    verify(ctx.get_presenter_mode)
+
+
+def test_presenter_mode_duration_as_string():
+    lib = MagicMock()
+    p = PropertyMock(return_value={'color': 'white', 'duration': '4s'})
+    type(lib).presenter_mode = p
+    ctx = LibraryComponent(lib)
+    verify(ctx.get_presenter_mode)
+
+
+def test_presenter_mode_duration_as_timedelta():
+    lib = MagicMock()
+    duration = timedelta(seconds=5)
+    p = PropertyMock(return_value={'color': 'black', 'duration': duration})
+    type(lib).presenter_mode = p
+    ctx = LibraryComponent(lib)
+    verify(ctx.get_presenter_mode)

